.buildkite/scripts/run_standard_models.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## remove failure for testing
+set -euo pipefail
+
+apt-get update
+apt-get install libsasl2-dev
+
+python3 -m venv venv
+. venv/bin/activate
+pip install --upgrade pip setuptools
+pip install -r integration_tests/requirements.txt
+mkdir -p ~/.dbt
+cp integration_tests/ci/sample.profiles.yml ~/.dbt/profiles.yml
+
+db=$1
+echo `pwd`
+cd integration_tests
+dbt deps ## Install all packages needed
+
+shift ## Skips the first argument (warehouse) and moves to only looking at the data model arguments
+
+for model in "$@" ## Iterates over all non warehouse arguments
+do
+    echo -e "\ncompiling "$model"\n"
+    cd dbt_packages/$model/integration_tests/
+    dbt deps
+    cp ../../../packages_ft_utils_override.yml packages.yml
+    dbt deps
+    if [ "$model" = "linkedin" ]; then
```

Comment:
Sure thing! The if statements are essentially needed because the packages that require a custom conditional are not structured in the same way as all the others. For example, we call the linkedin package `linkedin` however, all of the variables are named `linkedin_ads`. This shifts the structure of how the script code works because it by default will just use the package name to search for the schema variable. We need to make a small modification to ensure it works for linkedin then in this case. The same goes for the other packages with conditionals as well.